rs_client.py

```diff
@@ -1963,6 +1963,7 @@ def _cookie(request):
                 _r("    let reply = %s {", reply)
                 _r("        ptr: %s (self.conn.get_raw_conn(), self.cookie, &mut err)", func)
                 _r("    };")
+                _r("    if reply.ptr.is_null() { return Err(base::GenericError { ptr: err }) }")
```

Comment:
Uhm... This hands out a `GenericError` with a `NULL` pointer. Wouldn't that just cause crashes when this one is used?